manifests/prometheus/alerts.d/s3_bucket_count.yml

```diff
@@ -7,7 +7,7 @@
     name: S3BucketCountCloseToLimit
     rules:
       - alert: S3BucketCountCloseToLimit
-        expr: paas_aws_s3_buckets_count > ((aws_limits_s3_buckets)) / 100 * 80
```

Comment:
I think the intent was for this line not to change - `80 / 100 == 200 / 250`, so they're equivalent. It's just supposed to say `paas_aws_s3_buckets_count` is more than 80% of the configured limit.

Reply:
Probably `* 0.8` or `* 80/100` would be a more elegant way of expressing that mind you.

Reply:
Agreed, I did think it was an unnecessary way of getting to > 80% but went with the existing format. I'll switch it to  `* 0.8`